deploy CI job run failed on main branch

The `CUDA10_2-TORCH181` job error logs:

``` text
REMOTE_DEV_DOCKER: ***:latest-devel-cuda10.2
REMOTE_RUNTIME_DOCKER: ***:latest-runtime-torch1.8.1-cu102
RUNTIME_BASEIMAGE: nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04
GITHUB_PULL_REQUEST:
"docker build" requires exactly 1 argument.
See 'docker build --help'.
Usage:  docker build [OPTIONS] PATH | URL | -
Build an image from a Dockerfile
Error: Process completed with exit code 1.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -211,6 +211,7 @@ jobs:
         REMOTE_DEV_DOCKER: bladedisc:latest-devel-cuda10.2
         REMOTE_RUNTIME_DOCKER: bladedisc:latest-runtime-torch1.8.1-cu102
         RUNTIME_BASEIMAGE: nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04
+        DOCKERFILE: docker/runtime/Dockerfile.pytorch
       run: |
         set -e
         bash ./scripts/ci/deploy_pytorch_blade.sh

